### ISSUE TITLE
Optimal Learning Rate

### ISSUE THREAD

**Issue by XinDongol** [facebookresearch/MobileLLM#8]
In paper, the optimal learning rate is `2e-3`. 
In the `pretrain.sh`, the learning rate is set to `5e-4`. 
Could you please advise the best learning rate to train MobileLLM models?

**Comment by liuzechun** [comment]
Hi, thanks for the question. The optimal learning rate is 2e-3 when using 32 gpus (in the paper) and 5e-4 when using 8 GPUs (in the code). We linearly scale up/down the learning rate according to the total batch size (batch size * #GPUs). Hope this solves your question. Sorry for the late reply!

**Comment by XinDongol** [comment]
Thanks for replying. @liuzechun 
That makes sense. Nice work.
